frontend/src/components/v-ramp.vue

```diff
@@ -2,20 +2,36 @@
 .ramp
   template(v-if="tframe === 'commit'")
     template(v-for="(slice, j) in user.commits")
-      a.ramp__slice(
-        draggable="false",
-        v-on:click="rampClick",
-        v-for="(commit, k) in slice.commitResults.filter(commitResult => commitResult.insertions > 0)",
-        v-bind:href="getLink(commit)", target="_blank",
-        v-bind:title="getContributionMessage(slice, commit)",
-        v-bind:class="'ramp__slice--color' + getSliceColor(slice.date)",
-        v-bind:style="{\
-          zIndex: user.commits.length - j,\
-          borderLeftWidth: getWidth(commit) + 'em',\
-          right: ((getSlicePos(slice.date)\
-            + (getCommitPos(k, slice.commitResults.length))) * 100) + '%'\
-          }"
-      )
+      template(v-for="(commit, k) in slice.commitResults.filter(commitResult => commitResult.insertions > 0)")
+        a.ramp__slice(
+          draggable="false",
+          v-on:click="rampClick",
+          v-bind:href="getLink(commit)", target="_blank",
+          v-if="!isBrokenLink(getLink(commit))",
+          v-bind:title="getContributionMessage(slice, commit)",
+          v-bind:class="'ramp__slice--color' + getSliceColor(slice.date)",
+          v-bind:style="{\
+            zIndex: user.commits.length - j,\
+            borderLeftWidth: getWidth(commit) + 'em',\
+            right: ((getSlicePos(slice.date)\
+              + (getCommitPos(k, slice.commitResults.length))) * 100) + '%'\
+            }"
+        )
+        a.ramp__slice.broken-link(
```

Comment:
I think this design can be improved further. Currently, the difference between the two code segments are that:
1. The unsupported link has an additional `.broken-link` class.
2. The cursor is set to `default` for the unsupported link.

I think this can be improved such that we can use a ternary operator to determine if the `.broken-link` class is needed, and the `cursor: default` definition can be moved into that class.

Alternatively, if this design were to remain, I would suggest to remove the attributes that are not relevant for the click action (such as `v-on:click`, `target`, etc). This can make it more obvious that the broken link is not meant to be clicked on.